tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/base_enclave_info.h

```diff
@@ -27,6 +27,7 @@ class BaseEnclaveInfo {
         const std::string& persisted_sealed_data,
         const int num_of_enclaves);
     virtual ~BaseEnclaveInfo();
+    void terminate_enclave();
```

Comment:
Good to have consistent function naming convention.

Reply:
Could not follow.

Reply:
We have followed the convention of naming the function like `TerminateEnclave()` in cpp code. Good to keep the convention aligned. I am sure there are some places this convention is not followed, needs to be fixed.